policyengine_us/parameters/gov/states/ma/tax/income/credits/senior_circuit_breaker/eligibility/max_income.yaml

```diff
@@ -40,3 +45,5 @@ metadata:
       href: https://www.mass.gov/doc/2022-schedule-cb-circuit-breaker-credit/download
     - title: 2023 Schedule CB
       href: https://www.mass.gov/doc/2023-schedule-cb-circuit-breaker-credit/download
+    - title: Form 1 2024 Massachusetts Resident Income Tax - Annual Update of Circuit Breaker Tax Credit 
+      href: https://www.mass.gov/doc/2024-form-1-instructions/download#page=3
```

Comment:
add uprating metadata to this, max.yaml, and max_property_value.yaml (per statute)